resources/js/Pages/Benefits/Benefits.vue

```diff
@@ -41,6 +41,7 @@ function submitCreateBenefit() {
           type="button"
           class="inline-flex items-center py-3 px-4 text-sm font-medium leading-4 text-white bg-blumilk-600 hover:bg-blumilk-700 rounded-md border border-transparent focus:outline-none focus:ring-2 focus:ring-blumilk-500 focus:ring-offset-2 shadow-sm"
           @click="creating = true"
+          dusk="create-benefit-button"
```

Comment:
Please resolve this GitHub warning:

![image](https://github.com/Madoq/toby/assets/56546832/60ba1c16-1a7a-485e-8cb3-b9896efe21a8)